AnnoyancesFilter/sections/popups.txt

```diff
@@ -107,6 +107,7 @@ revouninstaller.com#$##popupModal { display: none !important; }
 revouninstaller.com#$#.modal-backdrop { display: none !important; }
 revouninstaller.com#$#body { overflow: visible !important; padding-right: 0 !important; }
 techmeme.com###nlsb
+indiamart.com###t0901_bewrapper
```

Comment:
`##.be-frmwrap` looks better and what about scrolling? I see it is blocked

Reply:
@Sergey-Lyapin, it seems, that i am not able to reproduce the popup anymore, event with AG off.